src/main/java/webserver/WebServer.java

```diff
@@ -9,6 +9,7 @@
 public class WebServer {
     private static final Logger logger = LoggerFactory.getLogger(WebServer.class);
     private static final int DEFAULT_PORT = 8080;
+    private static final UuidSessionManager uuidSessionManager = new UuidSessionManager();
```

Comment:
interface 를 만들었으니 인터페이스 이점을 좀 살려 보면 어떨까요?
```suggestion
    private static final SessionManager sessionManager = new UuidSessionManager();
```
이렇게 하고